### DISCUSSION TITLE
Are you still continuing this project?

### DISCUSSION THREAD

**Discussion by ThorstenBux** [creativelifeform/three-nebula#231] [General] [answered]
Hi,
I've noticed that the latest editor release is about 2 years old and the latest engine release is about one year old. Hence the question, are you still planning to continue this project?
We are potentially going to use the paid-for editor once it becomes available. But would need to confirm that this project is still going.

Cheers

**Comment by endquote** [comment]
I'm wondering this too -- the mac app just crashes for me.

**Reply by elugens** [reply]
> I'm wondering this too -- the mac app just crashes for me.

I think there is a Linux version that you should be able to use on your Mac...

**Reply by ThorstenBux** [reply]
When I try to download the Linux version I get
`Linux support is coming soon, please follow us on [twitter](https://twitter.com/getnebula) for the latest updates!` so I don't think there is a Linux version really

**Reply by cleversprocket** [reply]
I can get the app to work on my old Macbook Pro with an Intel chip. It crashes on my Mac Mini with an M1. I really hope this project gets some attention because its really cool and just what I needed. I would definitely pay for this app.

**Reply by Hootlook** [reply]
I'm a bit late to the party i think, but anyone knows where i can get the editor for windows ? The download link from the website is broken (502)..

**Reply by milewski** [reply]
> I'm a bit late to the party i think, but anyone knows where i can get the editor for windows ? The download link from the website is broken (502)..

Me too... have you figure out a mirror or something? how to get the app?

**Comment by nulref** [comment]
I was on a website called dddice.com. They have a version (not sure if its the most current one) available to download through their site.

https://docs.dddice.com/assets/files/nebula-v1.0.8-alpha-win64-c07b623342759250196826897a43bb84.zip

You can unzip that and install it but when you run it, its going to lock up as it looks for the update server, which no longer exists. Just turn off your internet before you start the Nebula UI and it works fine. You can reconnect to the internet once its running. You could probably reroute the server request in hosts, but I couldn't be bothered to find the url. I took me hours to figure out how to get this working. If anyone finds it, let us know.

**Reply by CelesteBloodreign** [reply]
Hi I am the dev of dddice - I have monkey patched the app and uploaded a version that doesn't need to have the internet off. You will get an error about the update server being missing - but you can use the app just fine. You can find it here:
https://docs.dddice.com/docs/guides/creating-themes/editor/particles/

If anyone has a mac build I would love to host that. And mr dev if you are there please contact me!

**Reply by rohan-deshpande** [reply, marked as answer]
I'm here. Sorry for inactivity, got very busy with life. I'm back though and have a bunch planned for this. Let me know what your top concerns / feature requests are.

Right now I'm working on updating this to work with modern three and then getting the designer into a better shape.

**Comment by CelesteBloodreign** [comment]
All I need is a working build of the Mac version. Also one that doesn’t die
if your update server goes down.
